opendatahub/quickstart/basic/README.md

```diff
@@ -85,10 +85,19 @@ The following `curl` examples demonstrate how to perform inference requests.
 
 **Curl test without authentication enabled**
 ~~~
-export HOST_URL=$(oc get route example-onnx-mnist -ojsonpath='{.spec.host}' -n ${TEST_MM_NS})
-export HOST_PATH=$(oc get route example-onnx-mnist -ojsonpath='{.spec.path}' -n ${TEST_MM_NS})
+export HOST_URL=$(oc get route example-onnx-mnist -ojsonpath='{.spec.host}' -n modelmesh-serving)
+
+export HOST_PATH=$(oc get route example-onnx-mnist -ojsonpath='{.spec.path}' -n modelmesh-serving)
+
+##If you give commands echo $HOST_URL and echo $HOST_PATH you should get a similar output to the following:
+echo $HOST_URL
+example-onnx-mnist-modelmesh-serving.apps.user.dev.datahub.redhat.com
+echo $HOST_PATH
+/v2/models/example-onnx-mnist
+
```

Comment:
this one is the same. please check [this doc](https://github.com/opendatahub-io/modelmesh-serving/blob/main/opendatahub/quickstart/common_docs/modelmesh-install.md)
Especially this part
~~~
source ../env.sh

cd opendatahub/quickstart/$QUICK_START_FOLDER # (for example, cd opendatahub/quickstart/basic)

./deploy.sh
~~~